MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableProtocol.c

```diff
@@ -552,13 +558,17 @@ AddTableToList (
     // could be updated by OS present agent. For example, BufferPtrAddress in
     // SMM communication ACPI table.
     //
-    ASSERT ((EFI_PAGE_SIZE % 64) == 0);
-    Status = gBS->AllocatePages (
-                    AllocateMaxAddress,
-                    EfiACPIMemoryNVS,
-                    EFI_SIZE_TO_PAGES (CurrentTableList->TableSize),
-                    &AllocPhysAddress
-                    );
+    if (IsFromHob){
+      AllocPhysAddress = (UINTN)Table;
+    } else {
+      ASSERT ((EFI_PAGE_SIZE % 64) == 0);

```

Comment:
Yes. This is an old pull request, pull request #5 . Acutally, Maurice mentioned it in pull request #6, and I have changed it and pull request #6 is already merged in, so we can't see it in pull request list. It seems like when I update the code, I should override the old pull request instead of creat a new one.

Reply:
I only see 1 pull request there from for_upstream branch. It looks your #6 is not shown in the pull request. yes, better use same branch to update the patch.